src/Middleware/HttpLogging/src/W3CLogger.cs

```diff
@@ -38,7 +38,7 @@ public void Log(string[] elements)
         _messageQueue.EnqueueMessage(Format(elements));
```

Comment:
One of the savings that was mentioned in the issue is that the temporary string from the formatting might be able to be avoided if the elements array is enqueued and the format is written directly to the stream writer via `W3CLoggerProcessor.WriteMessageAsync`. It would require a bit of refactoring of the underlying FileLoggerProcessor to store `string[]`s instead of strings and moving the format code.

Reply:
Sure !!. I've planned this change for a second PR (on the same issue). But I can work on it and add the change on this PR 🙂